regexes/client/hints/browsers.yml

```diff
@@ -266,3 +266,27 @@
 'ginxdroid.gdm': 'GinxDroid Browser'
 'on.browser': 'OnBrowser Lite'
 'com.pico.browser.overseas': 'PICO Browser'
+'com.cliqz.browser': 'Cliqz'
+'org.hola.prem': 'hola! Browser'
+'com.baidu.browser.inter': 'Baidu Browser'
+'org.torproject.torbrowser_alpha': 'Tor Browser'
+'com.ilegendsoft.mercury': 'Mercury'
+'com.apn.mobile.browser.coolpad': 'APN Browser'
+'com.apn.mobile.browser.infosonics': 'APN Browser'
+'net.dezor.browser': 'Dezor'
+'com.involta.involtago': 'Involta Go'
+'jp.ddo.pigsty.HabitBrowser': 'Habit Browser'
+'org.browser.owl': 'Owl Browser'
+'com.orbitum.browser': 'Orbitum'
+'com.appsverse.photon': 'Photon'
+'fr.agrange.bbbrowser': 'Keyboard Browser'
```

Comment:
do you have information about this browser?

I have come across keyboard identifiers with the name in the prefix, which in itself is not a browser.
any link or screenshot will be a confirmation that this is a browser.

Reply:
Do you mean the "Keyboard Browser"? It seems to be this one https://play.google.com/store/apps/details?id=fr.agrange.bbbrowser